tensorflow/python/ops/math_ops.py

```diff
@@ -3305,6 +3305,14 @@ def log_sigmoid(x, name=None):
   Specifically, `y = log(1 / (1 + exp(-x)))`.  For numerical stability,
   we use `y = -tf.nn.softplus(-x)`.
 
+  Usage Example:
+   >>> tensor1 = [0.0, 1.0, 2.0, 3.0, 4.0, 5.0]
+   >>> tensor2 = [2.0, 5.7]
+   >>> print(tf.math.log_sigmoid(tensor1, name='log_sigmoid_tensor1'))
+   tf.Tensor([-0.6931472  -0.31326166 -0.12692805 -0.04858733 -0.01814996 -0.00671535], shape=(6,), dtype=float32)
```

Comment:
Please correct the output format, as seen [here](https://www.tensorflow.org/community/contribute/docs_ref). Same for the next output.